addons/point_of_sale/static/src/app/store/combo_configurator_popup/combo_configurator_popup.xml

```diff
@@ -2,11 +2,14 @@
 <templates id="template" xml:space="preserve">
     <t t-name="point_of_sale.ComboConfiguratorPopup">
         <Dialog title="props.product.display_name" contentClass="'combo-configurator-popup'">
+            <div t-if="isArchivedProductSelected()" class="alert alert-warning mt-3">
+                <span>This combination does not exist.</span>
+            </div>
```

Comment:
can you not check from the start and disable the combinations that are not permitted, instead of allowing the user to click and then showing this message ?

Reply:
@vlst-odoo 
Values are getting disabled visually based on the selected values.
but the message is intended to inform users about the disabled options, helping to clarify why the restrictions.
When users click on a disabled option, there should be a clear message explaining the behavior.